documentation/book/assembly-upgrade.adoc

```diff
@@ -3,20 +3,83 @@
-Upgrading the version of {ProductName} is done by upgrading the Cluster Operator deployment to the new version.
+Later versions of {ProductName} provide support for newer versions of Kafka, so you need to upgrade {ProductName} _before_ you can upgrade to a higher supported Kafka version.
 
 ifdef::StrimziUpgrades[]
-include::proc-upgrading-the-cluster-operator-0-10-0-to-0-11-0.adoc[leveloffset=+1]
+*Kafka versions*
+
+The Kafka message format version and inter-broker protocol version are subject to change. As a result, the upgrade process involves making both configuration changes to existing Kafka brokers and code changes to client applications (consumers and producers). The following table shows the differences between the Kafka versions:
+
+include::snip-kafka-versions.adoc[leveloffset=+1]
+
+endif::StrimziUpgrades[]
+
+ifndef::StrimziUpgrades[]
+*Kafka versions*
+
+{ProductName} 1.0.0 and 1.1.0 are based on different versions of Apache Kafka:
+
+[options="header"]
+|=======================
+|{ProductName} version |Kafka version
+|1.0.0                 |2.0.0
+|1.1.0                 |2.1.1
+|=======================
+
+The Kafka message format version and inter-broker protocol version are subject to change. As a result, the upgrade process involves making both configuration changes to existing Kafka brokers and code changes to client applications (consumers and producers). The following table shows the differences between the Kafka versions:
+
+[options="header"]
+|=======================
+|Kafka version |Interbroker protocol version  |Log message format version | Zookeeper version
+|2.0.0         |2.0                           |2.0                        | 3.4.13
+|2.1.1         |2.1                           |2.1                        | 3.4.13
+|=======================
+
+Although Kafka 2.0.0 and 2.1.1 use the same version of Zookeeper, Red Hat recommends that you update your Zookeeper cluster to use the newest Zookeeper binaries before proceeding with the main {ProductName} 1.1.0 upgrade.
+endif::StrimziUpgrades[]
+
+*Message format version*
+
+When a producer sends a message to a Kafka broker, the message is encoded using a specific format. This is referred to as the message format version. You can configure a Kafka broker to convert messages from newer format versions to a given older format version, before the broker appends the message to the log.
+
+In Kafka, there are two different methods for setting the message format version:
+
+* The `log.message.format.version` property is set on Kafka brokers.
+
+* The `message.format.version` property is set on topics.
+
+The default value of `message.format.version` for a  topic is defined by the `log.message.format.version` that is set on the Kafka broker. You can manually set the `message.format.version` of a topic by modifying its topic configuration.
+
+In the upgrade tasks, we assume that the message format version is defined by the `log.message.format.version`.
+
+*Procedure outline*
```

Comment:
Please change this into a header, so that we can link to them.

Reply:
Changed